confignode/src/main/java/org/apache/iotdb/confignode/manager/load/cache/LoadCache.java

```diff
@@ -194,6 +223,21 @@ public Map<Integer, Pair<NodeStatistics, NodeStatistics>> updateNodeStatistics()
     return differentRegionGroupStatisticsMap;
   }
 
+  public Map<TConsensusGroupId, Pair<Integer, Integer>> updateRegionGroupLeader() {
+    Map<TConsensusGroupId, Pair<Integer, Integer>> differentRegionGroupLeaderMap =
+        new ConcurrentHashMap<>();
```

Comment:
why use `ConcurrentHashMap`? Will subscriber update this map concurrently?

Reply:
No. I use `HashMap` now.